CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+1.10.0-RC2 / 2020-05-07
```

Comment:
Add a 1.10.0 section, that includes all the items from RC1 and RC2

Reply:
Should i keep 1.10.0-RC1 and RC2 as well?